composer.json

```diff
@@ -86,10 +86,10 @@
         "ocramius/proxy-manager": "~0.4|~1.0",
         "egulias/email-validator": "~1.2",
         "symfony/security-acl": "~2.8|~3.0",
-        "phpdocumentor/reflection": "^1.0.7"
+        "phpdocumentor/reflection-docblock": "^3.0"
     },
     "conflict": {
-        "phpdocumentor/reflection": "<1.0.7"
+        "phpdocumentor/reflection-docblock": "<3.0"
```

Comment:
I don't think also it make sense here as the suggest is only in the component not in main composer. However it can help user to know which version of this library is supported, but maybe not it's place in the main composer.